src/redux/reducers/jewelry/action.tsx

```diff
@@ -25,6 +25,10 @@ export const fetchJewelry = (
     });
 };
 
+export const fetchJewelryById = async (id: string) => {
+  return await axios.get(`http://localhost:8081/jewelry/${id}`).then((data) => data.data);
```

Comment:
Move server URL to a constant so it can be easily changed.
If using `async/await` why using `.then`?